ingestion/src/metadata/ingestion/sink/elasticsearch.py

```diff
@@ -166,39 +166,26 @@ def __init__(
             ssl_context.check_hostname = False
             ssl_context.verify_mode = ssl.CERT_NONE
 
+        self.elasticsearch_client = Elasticsearch(
+            [
+                {"host": self.config.es_host, "port": self.config.es_port},
+            ],
+            http_auth=http_auth,
+            scheme=self.config.scheme,
+            use_ssl=self.config.use_ssl,
+            verify_certs=self.config.verify_certs,
+            ssl_context=ssl_context,
+            ca_certs=self.config.ca_certs,
+        )
         if self.config.use_AWS_credentials:
             credentials = boto3.Session().get_credentials()
             http_auth = AWS4Auth(
                 region=self.config.region_name,
                 service="es",
                 refreshable_credentials=credentials,
             )
-
-            self.elasticsearch_client = Elasticsearch(
-                [
-                    {"host": self.config.es_host, "port": self.config.es_port},
-                ],
-                http_auth=http_auth,
-                scheme=self.config.scheme,
-                use_ssl=self.config.use_ssl,
-                verify_certs=self.config.verify_certs,
-                ssl_context=ssl_context,
-                ca_certs=self.config.ca_certs,
-                connection_class=RequestsHttpConnection,
-            )
-
-        elif not self.config.use_AWS_credentials:
-            self.elasticsearch_client = Elasticsearch(
-                [
-                    {"host": self.config.es_host, "port": self.config.es_port},
-                ],
-                http_auth=http_auth,
-                scheme=self.config.scheme,
-                use_ssl=self.config.use_ssl,
-                verify_certs=self.config.verify_certs,
-                ssl_context=ssl_context,
-                ca_certs=self.config.ca_certs,
-            )
+            self.elasticsearch_client.http_auth = http_auth
+            self.elasticsearch_client.connection_class = RequestsHttpConnection
```

Comment:
I'd recommend to have this only when `use_AWS_credentials` is set as `true`. From the [official documentation](https://elasticsearch-py.readthedocs.io/en/7.x/transports.html), the default `urllib3` class is more performant and lightweight. Requests should only be used when custom-auth (using boto3, in this case, is required)